include/host_utils.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "host_queue.h"
+#include "item.h"
+
+namespace celerity {
+
+template <int Dims, typename Functor>
+void for_each_item(const partition<Dims> part, Functor func) {
```

Comment:
Since this is a bit of a drive-by API addition, could you add it to the (not yet existing) `[Unreleased]` section of the changelog, so we don't forget about it?

Reply:
Good idea!
I'll do that and also add the 2 new env variables. (Those aren't relevant for most users, but they are still part of the interface I guess and should be documented in the full changelog)